Dockerfile

```diff
@@ -5,27 +5,6 @@
 ##
 
 
-# Build project
```

Comment:
oh, is it the reason we checked in the `dist` folder?

Reply:
I think we should still keep the building step, it keeps the repo clean and at the same time make sure the dist folder is always generated from the source, what do you think?

Reply:
not sure, I think I picked this up while rebasing. @cjford ?

Reply:
Yeah this was a temporary experiment, after you branched it was removed via hard reset, if you rebase again on current origin/master it should go away.